test/unit/tasks/publish_finders_test.rb

```diff
@@ -0,0 +1,67 @@
+require "test_helper"
+require "rake"
+
+class PublishFindersRakeTest < ActiveSupport::TestCase
+  setup do
+    Rake::Task["finders:temp_unpublish_non_english_finders"].reenable
+  end
+
+  test "it publishes then marks as gone each non-English finder" do
+    Locale.stubs(:non_english).returns([
+      Locale.new(:cy),
+      Locale.new(:fr),
+    ])
+
+    Services.publishing_api.expects(:put_content).with("88936763-df8a-441f-8b96-9ea0dc0758a1", has_entries(
+                                                                                                 base_path: "/government/announcements.cy",
```

Comment:
I find these tests a bit noisy but I'm not super fussed given we're going to remove them fairly quickly (and we're actually good at removing code now✨)

Reply:
Yeah, I did think about DRY-ing them up, but as the code will only get run once then deleted, it didn't seem to be worth the effort.